regional_mom6/regional_mom6.py

```diff
@@ -810,7 +713,7 @@ def __init__(
             input_rundir.symlink_to(self.mom_run_dir.resolve())
 
     def __str__(self) -> str:
-        return json.dumps(self.write_config_file(export=False, quiet=True), indent=4)
+        return json.dumps(Config.save_to_json(export=False, quiet=True), indent=4)
```

Comment:
Do you need to pass `self` here? Also, `Config` doesn't have a `quiet` flag.

```suggestion
        return json.dumps(Config.save_to_json(self, export=False), indent=4)
```

Reply:
We do, thanks, I'll write a test case for this.